schlib/comparelibs.py

```diff
@@ -35,6 +35,7 @@ def ExitError( msg ):
 parser.add_argument("-v", "--verbose", help="Enable extra verbose output", action="store_true")
 parser.add_argument("--check", help="Perform KLC check on updated/added components", action='store_true')
 parser.add_argument("--nocolor", help="Does not use colors to show the output", action='store_true')
+parser.add_argument("--design_breaking_changes", help="Checks if there have been changes made that would break existing designs using a particular symbol.", action='store_true')
```

Comment:
Long options usually use hyphens to separate words, not underscores.  I'd rather see this option called `--design-breaking-changes`.  An option with this name would be stored as `args.design_breaking_changes` automatically, so you wouldn't have to change any other code to do this.